marketdata/rest.go

```diff
@@ -44,6 +44,11 @@ type ClientOpts struct {
 	RequestHost string
 }
 
+type request interface {
```

Comment:
[minor] I'd call this `cryptoRequest`

Reply:
[minor] Please move this to the top of the crypto section.

Reply:
Done.